app-office/Focuswriter/focuswriter-1.6.7.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="A QT Word Processor"
+DESCRIPTION="Focuswriter is an Open Source QT Word Processor"
+HOMEPAGE="https://gottcode.org/focuswriter/"
+COPYRIGHT="2008-2017 Graeme Gott"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/portableapps/files/Source/FocusWriter/focuswriter-1.6.7-src.tar.bz2"
+CHECKSUM_SHA256="bf78e596d7ed9de774bef5bbe75f1f0a01d626198fce6c8062a679936dfc3883"
+SOURCE_DIR="focuswriter-$portVersion"
+ADDITIONAL_FILES="focuswriter.rdef.in"
+
+ARCHITECTURES="x86_64 ?x86 ?x86_gcc2"
```

Comment:
The `ARCHITECTURES` order should be `?x86_gcc2 ?x86 x86_64` and as you are using `$secondaryArchSuffix` please add `SECONDARY_ARCHITECTURES` below line 12.